status-reporter/status_reporter/rabbitmq_reporter.py

```diff
@@ -13,17 +13,110 @@
 #  * See the License for the specific language governing permissions and
 #  * limitations under the License.
 
-from cloudify.cluster_status import CloudifyNodeType
+import json
 
-from .status_reporter import Reporter
+from requests import get
+from requests.exceptions import RequestException
+
+from cloudify.utils import LocalCommandRunner
+from cloudify.exceptions import CommandExecutionException
+from cloudify.cluster_status import (CloudifyNodeType, ServiceStatus,
+                                     NodeServiceStatus)
+
+from .status_reporter import Reporter, logger
+from .utils import get_systemd_services, get_node_status
+
+RABBITMQ_SERVICE_KEY = 'RabbitMQ'
+CA_PATH = '/etc/cloudify/ssl/rabbitmq-ca.pem'
+RABBITMQ_SERVICES = {'cloudify-rabbitmq.service': 'RabbitMQ'}
+RABBITMQ_URL = 'https://localhost:15671/api/healthchecks/node'
 
 
 class RabbitMQReporter(Reporter):
     def __init__(self):
         super(RabbitMQReporter, self).__init__(CloudifyNodeType.BROKER)
 
     def _collect_status(self):
-        return {}
+        services, statuses = get_systemd_services(RABBITMQ_SERVICES)
+        config = self._config('extra_config', None)
```

Comment:
Let change 'config' => credienitals\_config

Reply:
Also, the None can be removed it's the default:
`self._config.get('extra_config', {})`

Reply:
Oh now I see the `get` was missing